osgp/platform/osgp-adapter-domain-core/src/test/java/org/opensmartgridplatform/adapter/domain/core/application/services/DeviceInstallationServiceTest.java

```diff
@@ -0,0 +1,275 @@
+package org.opensmartgridplatform.adapter.domain.core.application.services;
```

Comment:
Formatting not ok, check your IDE settings

Reply:
Which part of the formatting is not okay?